examples/clients/simple-sampling-client/.env.example

```diff
@@ -0,0 +1 @@
+OPENAI_API_KEY=YOUR_OPENAI_API_KEY
```

Comment:
Can we rename this to `LLM_API_KEY` to not call out any specific model providers.

See simple chatbot example for inspiration how to provide an `LLMClient` to illustrate that any provider can be used: https://github.com/modelcontextprotocol/python-sdk/blob/c81ff64e94469c841010c1fef43cb298c6dc78d3/examples/clients/simple-chatbot/mcp_simple_chatbot/main.py#L215